.github/workflows/release-post-merge.yml

```diff
@@ -66,7 +80,7 @@ jobs:
         uses: ./.github/actions/certify-openshift-images
         with:
           repository: ${{ env.IMAGE_REPOSITORY }}
-          version: ${{ steps.tag.outputs.version}}
+          version: ${{ steps.tag.outputs.version }}
```

Comment:
shouldn't the new `{version}-certified` tag goes through the preflight certification process?

Reply:
I think the SHA is the same for both, but I fixed that just in case.